src/parsers/action-parser.js

```diff
@@ -39,7 +39,7 @@ export default class ActionParser {
   parse(obj = { cleaned: '', time: [] }) {
     let action = null;
 
-    if (obj.time === null) {
+    if (!obj.time || !obj.time.length) {
```

Comment:
This is quite unrelated to the global change, but I found this was quite inconsistent with the default value upthere. Not sure there is an actual bug because of this but I think this makes no harm.